addons/core/translations/en-us.yaml

```diff
@@ -56,7 +56,16 @@ onboarding:
       choose-path: Welcome to Boundary
       quick-setup: Quick setup
       manual: Manual
+      create-resource: Creating your first target
+      resource-setup-info: We’ll set up scopes & a host catalog for you
+      host-address: Host Address
+      target-port: Target Port
+      toggle-resource-info:  I have an IP address and port for my first host
     descriptions:
       choose-path: You’ve taken the first step towards making your infrastructure more secure. How do you want to get started?
       quick-setup: I want to see how Boundary works, help me set up my first scopes and targets.
       manual: I want to set up my first scopes and targets myself, starting with my first org.
+      create-resource: In Boundary, a “target” defines a server that your users will connect to.
+      resource-setup-info: Hosts are organized in host catalogs, which live in projects, which live in orgs. We will create these for you, and they can be edited or deleted later.
+      host-address: "Example: “192.168.0.1” or IPv6"
+      target-port: "Example: “22”"
```

Comment:
I believe this is how we were nesting the translations but it has changed. Can you confirm the latest commit on my "choose-path" branch is pulled into your branch? That should have the updated pattern.

Reply:
I couldn't find that branch. But this pr has the latest pattern from [main](https://github.com/hashicorp/boundary-ui/blob/main/addons/core/translations/en-us.yaml#L53).  do you have a separate branch with an updated pattern?

Reply:
Resolving comment after discussion.